deeppavlov/skills/aiml_skill/requirements.txt

```diff
@@ -0,0 +1 @@
+python-aiml==0.9.1
```

Comment:
This bothers me. When installing `deeppavlov` from `pip` user will not see this requirement.
I see a solution in creating a deeppavlov config for the aiml skill that will have a link on this requirement that will be moved to `/deeppavlov/requirements`